go.mod

```diff
@@ -49,7 +50,6 @@ require (
 	github.com/hashicorp/logutils v1.0.0 // indirect
 	github.com/hashicorp/terraform-exec v0.21.0 // indirect
 	github.com/hashicorp/terraform-json v0.22.1 // indirect
-	github.com/hashicorp/terraform-plugin-framework v1.11.0 // indirect
```

Comment:
This automatically happened when I upgraded the Go SDK. Leaving it for now as it's a no-op change.